Content.Server/GameTicking/GameTicker.RoundFlow.cs

```diff
@@ -160,6 +160,12 @@ public IReadOnlyList<EntityUid> LoadGameMap(GameMapPrototype map, MapId targetMa
             // whereas the command can also be used on an existing map.
             var loadOpts = loadOptions ?? new MapLoadOptions();
 
+            if (map.RandomOffset)
+                loadOpts.Offset = _robustRandom.NextVector2(map.MaxRandomOffset);
```

Comment:
Why not check if MaxRandomOffset > 0 and ditch the bool?

Reply:
I was thinking of yaml maintainability but it probably is a non-issue so I'll do that.